ansible/roles/nessus/files/nessus_base.py

```diff
@@ -120,6 +122,7 @@ def __make_request(self, target, method, payload=None, files=None):
                     headers=headers,
                     params=payload,
                     verify=VERIFY_SSL,
+                    timeout=REQUEST_TIMEOUT,
```

Comment:
Should we sort these named arguments alphabetically here and elsewhere below?

Reply:
We can. I always err on the side of conservative when it comes to changes in a Lineage PR so I just plopped it at the end.

Reply:
@mcdonnnj Any update here?

Reply:
@dav3r I'd like to do the sorting in a separate PR because I'd rather be comprehensive about it and that's outside the scope of this PR (in my opinion). I did test that the timeout value works ok when redeploying an instance.